src/renderer/components/query.jsx

```diff
@@ -68,11 +68,6 @@ export default class Query extends Component {
   }
 
   componentDidMount() {
-    this.refs.queryBoxTextarea.editor.on(
```

Comment:
I'm not sure what the purpose of this was, but by removing it I removed a duplicate dispatch of the `UPDATE_QUERY` action on every key press, and solved an issue where I wasn't loading the correct `query.query` into the `AceEditor` pane per tab.